public/app/core/components/UserEdit/state/actions.ts

```diff
@@ -0,0 +1,65 @@
+import { getBackendSrv } from '@grafana/runtime';
+import { OrgUser, ThunkResult } from 'app/types';
+
+export enum ActionTypes {
+  LoadUser = 'LOAD_USER',
+}
+
+interface LoadUserAction {
+  type: ActionTypes.LoadUser;
+  payload: OrgUser;
+}
+
+const userLoaded = (user: OrgUser) => ({
+  type: ActionTypes.LoadUser,
+  payload: user,
+});
+
+export type Action = LoadUserAction;
+
+export function loadUser(id?: number): ThunkResult<any> {
```

Comment:
could this be moved to the page level component instead (ie not redux)? that way we do not need the clear action. You ad a comment that this was used by the permissions component, but can't you just send in the user object to that component via props?